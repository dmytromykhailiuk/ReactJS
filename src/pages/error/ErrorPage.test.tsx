import { shallow } from 'enzyme';
import React from 'react';
import toJson from 'enzyme-to-json';
import { ErrorPage } from '.';

describe('ErrorPage', () => {
  it('should match first snepshot', () => {
    const wrapper = shallow(<ErrorPage navigateToHome={() => {}} />);
    expect(toJson(wrapper)).toMatchSnapshot('first');
  });
});
