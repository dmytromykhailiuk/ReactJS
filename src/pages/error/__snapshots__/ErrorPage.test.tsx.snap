// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPage should match first snepshot: first 1`] = `
<div
  className="error-page"
>
  <h2
    className="error-page__title"
  >
    Page Not Found
  </h2>
  <div
    className="error-page__error-number"
  >
    404
  </div>
  <div
    className="error-page__button"
  >
    <Button
      onButtonClicked={[Function]}
      style={
        Object {
          "width": "300px",
        }
      }
      type="secondary"
    >
      GO BACK TO HOME
    </Button>
  </div>
  <div
    className="error-page__logo"
    onClick={[Function]}
  >
    <Logo />
  </div>
</div>
`;
